src/scripts/exclusion_term_expansion/exclusion_term_expansion.py

```diff
@@ -0,0 +1,312 @@
+"""Takes in the full ontology and the exclusions tsv and extracts a simple list of terms from it.
+Outputs two files:
+1. a simple list of terms (config/ONTO_NAME_term_exclusions.txt)
+2. a simple two column file with the term_id and the exclusion reason
+
+# Resources
+- GitHub issue: https://github.com/monarch-initiative/mondo-ingest/issues/22
+- ICD10CM exclusions, online: https://docs.google.com/spreadsheets/d/1cZPUReTl34Vu2a03tm2921ehARiIpp4fjeQfS27XzKA/edit#gid=1119821904
+- Ophanet/ORDO exclusions, online: https://docs.google.com/spreadsheets/d/16ftBJ8mYqEvSEVNi1tGxIlDfL88vYrs7tSUjSnGqrBw/edit#gid=0
+
+TODO's
+  - todo: Do we need to worry about edge case where exclusion table is empty? (e.g. we include such a table for all
+     ontologies, but there exists 1 where we don't exclude anything.
+  - todo: later: see below: #x1: regex results: would be good to filter out by IRI or prefix that is not from source
+     ontology, but how to determine the appropriate prefix just based on the 2 file params, but could have edge cases.
+  - todo: later: see below: #x2
+  - todo: later: see below: #x3
+  - todo: later: see below: #x4: Switch from SPARQL to OAK. (SqliteImpl() is faster than SparqlWrapper (which uses rdflib)
+  - todo: later: When constructing exclusion TSV: bug: ICD10CM:C7A-C7A reported as term, but should be ICD10CM:C7A
+  - todo: later: QA: possible conflicts in icd10cm_exclusions.tsv: Add a check to raise an error in event of a parent
+     class having `True` for `exclude_children`, but the child class has `False`.
+"""
+import os
+import subprocess
+from argparse import ArgumentParser
+from typing import Dict, List, Union
+
+from jinja2 import Template
+import pandas as pd
+
+
+# Vars
+# # Config
+CONFIG = {
+    'use_cache': False,
+    'include_superclass': False,  # is this the correct name for what this does: * property path qualifier
+    'save': True,  # save dataframes to disk
+    'regex_prefix': 'REGEX:'  # used to know which rows in exclusion TSVs are regex pattern searches
+}
+
+# # Static
+THIS_SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+CACHE_DIR = os.path.join(THIS_SCRIPT_DIR, 'cache')
```

Comment:
The universal "cache" dir in all ODK repos is:

```
src/ontology/tmp
```

Reply:
Done! Also changed for my other script.